Application/EdFi.Ods.SandboxAdmin/Initialization/InitializationSettingsFactory.cs

```diff
@@ -0,0 +1,54 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Configuration;
+using System.Linq;
+
+namespace EdFi.Ods.SandboxAdmin.Initialization
+{
+    /// <summary>
+    /// Provides initialization settings from web.config or app.config
+    /// </summary>
+    public class InitializationSettingsFactory : IInitializationSettingsFactory
```

Comment:
I think this is not necessary, i know we had a ticket to use the new format a while back

Reply:
removed